VERSION.cmake

```diff
@@ -1,14 +1,14 @@
 SET( VERSION_MAJOR "0")
 SET( VERSION_MINOR "5" )
-SET( VERSION_PATCH "1" )
+SET( VERSION_PATCH "2" )
 SET( VERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}${GIT_SHA1_VERSION}" )
 
 ##### This is need for the libyui core, ONLY.
 ##### These will be overridden from exports in LibyuiConfig.cmake
 # Currently you must also change so_version in libyui.spec
 # *and also in **all** other* libyui-*.spec files in the other repositories.
 # Yes, such a design is suboptimal.
-SET( SONAME_MAJOR "12" )
+SET( SONAME_MAJOR "0" )
```

Comment:
Shouldn't it be 13?

Reply:
No, for most of those dependent packages it doesn't make any sense to have it there at all; only for the master package.

This SO version at this place is only misleading. I found myself changing it only here several times, overlooking the SO version in the spec file which is really what needs changing.